tests/csm/rest/test_iam_users.py

```diff
@@ -4541,25 +4540,103 @@ def test_42270(self):
         count = resp_dict["count"]
         last_uid = resp_dict["marker"]
         assert_utils.assert_equals(count, 5, "Entries not returned as expected")
-
         counter = 0
-        for user in users_list:
-            if user in get_user_list:
-                self.log.info("%s user is listed in response", user)
-                counter += 1
-                users_list.pop(user)
+        flag = True
+        while flag:
             self.log.info("Step 3: Get next two entries")
             resp_new = self.csm_obj.list_iam_users_rgw(max_entries=2, marker=last_uid)
             assert_utils.assert_equals(resp_new.status_code, HTTPStatus.OK, "Status check failed")
             resp_new_dict = resp_new.json()
             count_new = resp_new_dict["count"]
-            get_user_list.append(resp_new_dict["users"])
-            if resp_new_dict["count"] == 0:
-                break
+            if count_new == 0:
+                flag = False
             else:
+                get_user_list.append(resp_new_dict["users"])
                 last_uid = resp_new_dict["marker"]
                 assert_utils.assert_equals(count_new, 2, "Entries not returned as expected")
+        for user in users_list:
+            if user in get_user_list:
+                self.log.info("%s user is listed in response", user)
+                counter += 1
+                users_list.pop(user)
         self.log.info("User list from GET response: %s", get_user_list)
         assert_utils.assert_equals(counter, self.csm_conf["common"]["num_users"],
                                    "Did not get all users")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42271')
+    def test_42271(self):
```

Comment:
Can we have test name instead of just test ID

Reply:
In CSM we have been using IDs only. Easier to run if there is CI/CD failure..

Reply:
ok